src/lib/containers/__tests__/works_for_you-tests.tsx

```diff
@@ -99,21 +99,8 @@ interface NotificationsResponse {
                 }
               }
             }
-          },
-          {
-            node: {
-              artists: string
-              date: string
-              message: string
-              artworks: [{ title: string }, { title: string }]
-              image: {
-                resized: {
-                  url: string
-                }
-              }
-            }
           }
-        ]
+        >
       }
```

Comment:
@mennenia your interface here was wrong, you're actually putting an array inside `edge` - I've turned it into an array 👍